tests/fixtures.py

```diff
@@ -535,6 +535,14 @@ def testerchain(_testerchain) -> TesterBlockchain:
     yield testerchain
 
 
+@pytest.fixture(scope='module')
+def _mock_testerchain() -> MockBlockchain:
```

Comment:
Moving this permanently to `tests.fixtures` from `test.integrations.conftest`?

Reply:
Yes, it can be useful for unit tests, which is what we do in this PR.